matchers/python/todomatcher.go

```diff
@@ -2,30 +2,64 @@ package python
 
```

Comment:
The comment on the previous todo matcher applies for all of them.

Not going to repeat the same stuff all over the place.